src/components/DashboardHeader.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { Component }  from "react";
+import React, { Component , useState }  from "react";
```

Comment:
it looks like you're missing prettier extension in vscode (code looks unformatted), isn't it ? :)
this may cause problems during merges when collaborating with other team members, 
may you check that you've format on save (with prettier) active in your editor?